app/src/main/java/com/kickstarter/viewmodels/SurveyHolderViewModel.kt

```diff
@@ -34,12 +30,9 @@ interface SurveyHolderViewModel {
         fun startSurveyResponseActivity(): Observable<SurveyResponse>
     }
 
-    class ViewModel(@NonNull environment: Environment) :
-        ActivityViewModel<SurveyViewHolder>(environment),
-        Inputs,
-        Outputs {
+    class ViewModel : androidx.lifecycle.ViewModel(), Inputs, Outputs {
```

Comment:
i dont think we're supposed to inherit from android `ViewModel` since this is only a view holder and it doesn't have a dedicated activity/fragment and therefore no attachment lifecycle.

Reply:
Good call, dont think it needs it so ill remove it!